tests/transaction/test_get_transaction.py

```diff
@@ -0,0 +1,172 @@
+"""
+Provide tests for command line interface's get transaction command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_transaction():
+    """
+    Case: get a transaction by id.
+    Expect: transaction is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'transaction',
+        'get',
+        '--id',
+        'c13fff007b5059ea0f95fc0dc0bdc897ef185b1e1187e355f3b02fb0aad515eb'
```

Comment:
Move to the separated variable. It is a bit harder to read one string as two lines.

Reply:
@dmytrostriletskyi, done.